Gemfile

```diff
@@ -23,6 +23,9 @@ gem 'mysql2'
 # for observing records
 gem 'rails-observers'
 
+# for rating
+gem 'ratyrate', github: 'wazery/ratyrate', branch: 'master'
```

Comment:
Using the gem from master:
![ratyrate_gem](https://cloud.githubusercontent.com/assets/14155445/24654436/3ed1f7a8-1957-11e7-99c6-33416ad07b61.png)

Using the released version:
![rayrate](https://cloud.githubusercontent.com/assets/14155445/24654438/3ee9df30-1957-11e7-85b2-c51052a78498.png)

The function to show current_user's vote renders only a single star in the released version which has been fixed in master but a new version hasn't been released yet.